packages/sw-build/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "sw-build",
+  "version": "0.0.8",
+  "description": "This module can be used to generate file manifest or service worke, that can be used with sw-lib.",
```

Comment:
service worker is missing the tail end of it. I'd also drop in an `a` before file here.